### PR TITLE
N16: ggez

### PR DESCRIPTION
**Pull request by icefoxen**
What the heck, I have no idea why this is dragging a bunch of other commits with it.  I nuked and re-made my fork and branch from this repo and it's still doing it.

It's been one of those days.

Part of #365

### CODE REVIEW

File: content/posts/newsletter-016/index.md [hunk 1]
```diff
@@ -67,6 +67,28 @@ If needed, a section can be split into subsections with a "------" delimiter.
...
+![ggez bunnymark - 60,000 buns at 60 FPS](ggez-bun.png)
+
+[ggez] is a 2D game framework inspired by Love2D.  We're chugging
+along getting everything prepared for a 0.6 release at the end of 2020,
+and a lot of work has been done in the last month:
+
+ * All major dependencies have been updated, including a long-overdue
+   update to `winit` 0.23
+ * Over a dozen pull requests have been merged, large and small
+ * A bunch of old issues have been cleaned up
```
Comment (17cupsofcoffee) [inline content/posts/newsletter-016/index.md:81]:
```suggestion
- All major dependencies have been updated, including a long-overdue
  update to `winit` 0.23
- Over a dozen pull requests have been merged, large and small
- A bunch of old issues have been cleaned up
```

**Review by 17cupsofcoffee — CHANGES_REQUESTED** [review]
I have *no* clue how this branch has gotten itself so confused - should be able to just cherry pick it to merge though, so not the end of the world 😄 

Text looks good to me, just one minor formatting thing that's tripping up the linter:

**Review by 17cupsofcoffee — APPROVED** [review]

**Review by ozkriff — APPROVED** [review]
:+1: (made a few small style/fmt tweaks)

**Comment by ozkriff** [comment]
Thanks, merged!

As for the other commits - force pushing the `source` branch from the upstream repo to your fork's `source` should help.

**Pull request merged**